biomodels/README.md

```diff
@@ -0,0 +1,62 @@
+ 
+ ```
+## models2pathways.tsv example
+BIOMD0000000498 R-HSA-917977    8.191346134722721E-7    https://reactome.org/PathwayBrowser/#/R-HSA-917977      Transferrin endocytosis and recycling   IEA     Homo sapiens
+BIOMD0000000287 R-HSA-69563     6.066213418698396E-6    https://reactome.org/PathwayBrowser/#/R-HSA-69563       p53-Dependent G1 DNA Damage Response    IEA     Homo sapiens
+BIOMD0000000287 R-HSA-2559586   6.066213418698396E-6    https://reactome.org/PathwayBrowser/#/R-HSA-2559586     DNA Damage/Telomere Stress Induced Senescence   IEA     Homo sapiens
+```
+
+For the purposes of this module, the first two columns are the most important. The first column contains BioModels identifiers, and the second column contains Reactome Pathway identifiers that have have been connected to them by biomodels-mapper. For each line in `models2pathways.tsv`, the corresponding Pathway instance in Reactome's relationship database will receive a CrossReference that points to the biomodel identifier. If the biomodel identifier does not already have a database instance, it will be created before being slotted into the Pathway instance as an instance of type `DatabaseIdentifier`. All biomodels identifier instances are connected to the sole 'BioModels Database' `ReferenceDatabase` object. 
+ 
+## Logging
+ 
+This application will log to a file under `./logs/biomodels-{timestamp}.log` and `./logs/biomodels-{timestamp}.err` for warnings and errors. This file will have the same content as the console (stdout/stderr) except without the log4j prefix. 
+ 
+## Configuration
+
+The configuration file is set up like a typical properties file, containing database adaptor information and the database ID of your person instance:
+
+```
+### Sample configuration file with properties
+release.database.host=mySQLHost
+release.database.user=mySQLUsername
+release.database.password=mySQLPassword
+release.database.port=3306
+release_current.name=release_current
+personId=123456
+```
+Once this has been completed and saved somewhere on the server, you should be able to run the biomodels insertion step.
+  
+## Compiling & Running
+
+This is a Java application which requries a Java 8+ environment. You will need maven and a full JDK to compile.
+
+To compile the application, run this command:
+
+```
+$ mvn clean compile package
+```
+
+If this is successful, you should see a JAR file in the `target` directory, with a name like `biomodels-VERSION_NUMBER-jar-with-dependencies.jar`. This is the file you will run.
+
+For example, for version "1.0-SNAPSHOT", execute this command to run the program:
+```
+$ java -jar target/biomodels-1.0-SNAPSHOT-jar-with-dependencies.jar ./config.properties path/to/models2pathways.tsv
+```
+
+If no properties file is specified as the first argument, the program will look in src/main/resources/ for config.properties. Similarly for models2pathways.tsv, unless otherwise specified it will assume it exists in src/main/resources.
```

Comment:
It's a little ambiguous here what will happen if the models2pathways.tsv file path is provided as the first and only argument.  Could the docs be updated to specify what happens and/or if this shouldn't be done?

As a separate GitHub issue or PR, maybe we can look into standardizing our data-release-pipeline jar artifacts to use named arguments.  It might even be a method or class we can create (for parsing arguments) in release-common-lib so we can have a standard that can be re-used.

Reply:
Created an issue for implementing arguments: https://github.com/reactome/data-release-pipeline/issues/101

Reply:
I also updated the language of the README